src/components/TransactionItemComponent/DataCeils/DateCellProps.ts

```diff
@@ -0,0 +1,9 @@
+import type Transaction from '@src/types/onyx/Transaction';
+
+type DataCellProps = {
```

Comment:
Type name doesn't match file name. File is called `DateCellProps` which is confusing 😅